packages/ui/core-components/src/lib/atoms/inputs/button-group/ButtonGroup.svelte

```diff
@@ -53,7 +53,7 @@
 	const { results, update } = buildReactiveInputQuery(
 		{ value, data, label, order, where },
 		`ButtonGroup-${name}`,
-		$page?.data?.data[`ButtonGroup-${name}`]
+		$page?.data?.data[`ButtonGroup-${name}_data`]
```

Comment:
I'm not super thrilled about this needing to have `_data` on here, but can fix that later

Reply:
Maybe we add something like `static setSSRData` on `Query`?

Reply:
I think we leave it for now, but I was speaking specifically about `InputName_data` instead of `InputName`